### PR TITLE
Mirror stdout + stderr to file

### PR DESCRIPTION
**Pull request by multimokia**
Even with [`0e848db`](https://github.com/Monika-After-Story/MonikaModDev/blob/0e848dbdaace268e8e381b2521c557f4f414a255/.github/workflows/mas_check.yml), we still didn't properly report errors in the GHActions script

~~using `tee` I mirrored the output from lint to stdout and to file so it's reported properly.~~

I modified the script to no longer use the lint parser, but instead run through `grep` with an inverted match criteria to find the lines referring to images

### CODE REVIEW

**Comment by multimokia** [comment]
Hmm we need a way of fixing that issue of all the missing images again

File: .github/workflows/mas_check.yml [hunk 1]
```diff
@@ -65,13 +65,10 @@ jobs:
       - name: rpy lint
         run: |
           cd renpy
-          ./renpy.sh "../mas0105/" lint &> renpy_output
-          python ../tools/renpy_lint_parser.py
-          cat renpy_output_clean
+          ./renpy.sh "../mas0105/" lint | grep -E -v "^$|Could not find image \(monika [0-9][^[:space:]]+ corresponding to attributes on say statement\.|'monika [0-9][^[:space:]]' is not an image\.|The image named 'monika [0-9][^[:space:]]+ was not declared\."
```
Comment (ThePotatoGuy) [inline .github/workflows/mas_check.yml:68]: This regex is covers too many cases. The previous one was very specific in catching forms of `monika #ccc+` (where `c` is any sprite code), `#c`, and specific other numbers/images. This is so an actual invalid code like `monika 1ef` would still appear in the log.

**Review by ThePotatoGuy — CHANGES_REQUESTED** [review]

File: .github/workflows/mas_check.yml (hunk 1 above)
Comment (Booplicate) [reply .github/workflows/mas_check.yml:68]: We're only hiding renpy lint reports, won't our sprite checker messages still pass through?

Comment (multimokia) [reply .github/workflows/mas_check.yml:68]: Yeah our spritechecker messages should still be pushed through since this only checks for renpy lint output

Comment (ThePotatoGuy) [reply .github/workflows/mas_check.yml:68]: that appears to be correct

**Review by ThePotatoGuy — APPROVED** [review]

**Review by ThePotatoGuy — APPROVED** [review]

**Pull request merged**